### DISCUSSION TITLE
ICPC Quantum Computing Challenge - Description and Link

### DISCUSSION THREAD

**Discussion by alphaX86** [EddieHubCommunity/support#904] [General] [answered]
For the first time, ICPC is hosting a Quantum Computing challenge at Feb 24. The event is sponsored by IBM Quantum and all are invited to participate! The materials are provided (Qiskit notebooks, Guides) and it's a 4 day event. 

If you're interested to hop in, here's the LinkedIn post

**LinkedIn**: https://www.linkedin.com/feed/update/urn:li:activity:6763515694844833792

**Form link**: https://challenges.quantum-computing.ibm.com/icpc

If you're interested to apply, you're welcome! 😄 Thanks!

**Comment by alphaX86** [comment, marked as answer]
Link posted and it's done for sharing the details about the event

**Reply by eddiejaoude** [reply]
Thanks for sharing 🤓
